css/styles.css

```diff
@@ -0,0 +1,720 @@
+:root {
+  --color-primary: #272a31;
+  --color-secondary: #ec5242;
+  --color-body: #515151;
+  --color-lightgray: #d3d3d3;
+  --color-lightergray: #fbfbfb;
+  --color-white: #fff;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+/* Typography */
+body {
+  color: var(--color-body);
+  font-family: "Lato", sans-serif;
+  font-size: 0.75rem;
+  font-weight: 400;
+  line-height: 1.5;
+}
```

Comment:
- [ ] Please kindly adjust the left and right padding of your app in accordance to the given template. 👍 Please don't forget to place the logo properly as well.

## Behance Design
<img width="454" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225626827-b1582ccd-37de-4991-b36e-cc5eb1edc1a4.png">

## Your Design
<img width="948" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225627320-2af53d91-f311-4f56-b597-2a07e36bc75c.png">